scripts/src/report/report.py

```diff
@@ -277,6 +279,21 @@ def _load_taxonomies(hits):
     }
 
 
+def _load_taxonomies_bold(hits):
+    return {
+        hit['accession']: {
+            "kingdom": hit.get("taxonomy", {}).get("kingdom", ""),
+            "phylum": hit.get("taxonomy", {}).get("phylum", ""),
+            "class": hit.get("taxonomy", {}).get("class", ""),
+            "order": hit.get("taxonomy", {}).get("order", ""),
+            "family": hit.get("taxonomy", {}).get("family", ""),
+            "genus": hit.get("taxonomy", {}).get("genus", ""),
+            "species": hit.get("taxonomy", {}).get("species", "")
```

Comment:
This is very repetitive. You could clean it up a bit with a dictionary comprehension:

```py
def _load_taxonomies_bold(hits):
    return {
        hit['accession']: {
            key: hit.get("taxonomy", {}).get(key, "")
            for key in (
                "kingdom",
                "phylum",
                "class",
                "order",
                "family",
                "genus",
                "species",
            )
        }
        for hit in hits if 'accession' in hit
    }
```